### ISSUE TITLE
Have you abandoned this project?

### ISSUE THREAD

**Issue by BugsWorld** [DreadedLama/Simple-Dialer#16]
I'm still using it on a daily as my main dialer app but It's starting to age. Wondering if you will ever merge with latest updates?

**Comment by DreadedLama** [comment]
Yes, I have decided to stop working on this project for personal reasons. Thanks for all the love and support shown for my fork :)
